package.json

```diff
@@ -17,9 +17,9 @@
     "banner": "cd bin/banner && npm install --silent && npm run -- banner"
   },
   "dependencies": {
-    "@docusaurus/core": "^3.1.1",
-    "@docusaurus/plugin-ideal-image": "^3.1.1",
-    "@docusaurus/preset-classic": "^3.1.1",
+    "@docusaurus/core": "^3.3.2",
+    "@docusaurus/plugin-ideal-image": "^3.3.2",
+    "@docusaurus/preset-classic": "^3.3.2",
```

Comment:
Bumped to `docusaurus>3.2.0` to benefit from https://github.com/facebook/docusaurus/pull/9581 which allows for local author images.